packages/react-core/src/components/Timestamp/__tests__/Timestamp.test.tsx

```diff
@@ -36,12 +36,31 @@ test('Renders with current date by default with default formatting', () => {
   expect(screen.getByText(new Date().toLocaleString())).toBeInTheDocument();
 });
 
+test('Renders with correct datetime attribute with current date by default', () => {
+  render(<Timestamp />);
+  // Because there could be a .001 ms difference in the expected and received datetime value,
+  // we want an ISO value without the ms to expect as the datetime value.
+  const isoDateWithoutMS = new Date().toISOString().split('.')[0];
+
+  expect(screen.getByText(new Date().toLocaleString())).toHaveAttribute(
+    'datetime',
+    expect.stringMatching(isoDateWithoutMS)
+  );
+});
+
 test('Renders passed in date with default formatting', () => {
   render(<Timestamp date={new Date(2022, 0, 1)} />);
 
   expect(screen.getByText('1/1/2022, 12:00:00 AM')).toBeInTheDocument();
 });
 
+test('Renders with correct datetime attribute when date is passed in', () => {
+  const passedDate = new Date(2022, 0, 1);
```

Comment:
Is the user timezone going to affect these tests? @gitdallas, is there a way we can easily recognize the sorts of tests you had issues with before?

Reply:
the test that fails for me without TZ=EST is "Renders with only date when dateFormat is passed in"
<img width="752" alt="image" src="https://user-images.githubusercontent.com/5322142/195433868-c715a4bd-2ddd-48b8-817e-795f7dbf81de.png">

Reply:
I believe I had formatted the passed in string for those few tests due to the wonkiness that was occurring when running the tests (the tests would pass locally but fail when ran on Github).

I could update those few tests so that instead of `new Date('...EST')` the format is similar to other tests, e.g. `new Date(2022, 0, 1)`. @gitdallas do you feel that would be a viable option?